src/Parsers/ParserCreateQuery.cpp

```diff
@@ -210,6 +223,14 @@ bool ParserTablePropertiesDeclarationList::parseImpl(Pos & pos, ASTPtr & node, E
             indices->children.push_back(elem);
         else if (elem->as<ASTConstraintDeclaration>())
             constraints->children.push_back(elem);
+        else if (elem->as<ASTIdentifier>() || elem->as<ASTFunction>())
+        {
+            if (primary_key)
+            {
+                throw Exception("Multiple primary keys are not allowed.", ErrorCodes::LOGICAL_ERROR);
```

Comment:
`LOGICAL_ERROR` identifies error unrecoverable by the user. I think `BAD_ARGUMENTS` will be better.

Reply:
Fixed. Thanks.